### PR TITLE
feat(kit): `TuiInputPhoneInternational` has separator customization

### PR DESCRIPTION
**Pull request by nezar-khamidekh**
Closes https://github.com/taiga-family/taiga-ui/issues/9518

### CODE REVIEW

File: projects/demo/src/modules/components/input-phone-international/index.html [hunk 1]
```diff
@@ -190,6 +190,22 @@
                 at the right side of the textfield.
             </ng-template>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="customize-with-separator"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-phone-international/index.html:195]:
```suggestion
            id="customize-separator"
```

File: projects/demo/src/modules/components/input-phone-international/index.html [hunk 2]
```diff
@@ -190,6 +190,22 @@
...
+            heading="Customize with separator"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-phone-international/index.html:196]:
```suggestion
            heading="Customize separator"
```

File: projects/kit/components/input-phone-international/input-phone-international.options.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ export interface TuiInputPhoneInternationalOptions {
     readonly countries: readonly TuiCountryIsoCode[];
     readonly countryIsoCode: TuiCountryIsoCode;
     readonly metadata: Observable<MetadataJson> | Promise<MetadataJson>;
+    readonly separator?: string;
 }
 
 export const TUI_INPUT_PHONE_INTERNATIONAL_DEFAULT_OPTIONS: TuiInputPhoneInternationalOptions =
```
Comment (nsbarsukov) [inline projects/kit/components/input-phone-international/input-phone-international.options.ts:15]: I think, we add `separator: '-'` to `TUI_INPUT_PHONE_INTERNATIONAL_DEFAULT_OPTIONS`

**Review by nsbarsukov — APPROVED** [review]
Thanks for your contribution!

File: projects/kit/components/input-phone-international/input-phone-international.options.ts (hunk 3 above)
Comment (nezar-khamidekh) [reply projects/kit/components/input-phone-international/input-phone-international.options.ts:15]: can I add it?

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**